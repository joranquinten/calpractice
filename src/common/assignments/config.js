export const operators = [
  { complexity: 1, operator: "+" },
  { complexity: 2, operator: "-" },
  { complexity: 3, operator: "*" },
  { complexity: 4, operator: "/" }
];

export const ranges = [
  { complexity: 1, low: 0, high: 10 },
  { complexity: 2, low: 0, high: 100 }
];
