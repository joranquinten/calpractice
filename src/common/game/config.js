export const parameters = {
  numberOfAssignments: 10,
  numberOfLives: 3,
  maxTimePerRound: 30000
};
