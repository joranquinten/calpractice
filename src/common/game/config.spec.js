import { parameters } from "./config";

describe("Game config", () => {
  it("should expose the default configuration", () => {
    expect(parameters).toMatchSnapshot();
  });
});
