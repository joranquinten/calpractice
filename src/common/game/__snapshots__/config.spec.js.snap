// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game config should expose the default configuration 1`] = `
Object {
  "maxTimePerRound": 30000,
  "numberOfAssignments": 10,
  "numberOfLives": 3,
}
`;
